requirements.txt

```diff
@@ -1,13 +1,15 @@
-torch==1.2.0
-torchvision==0.2.2
-tensorboardX==1.2
+torch>=1.2
+torchvision>0.2
+tensorboardX>=1.2
 numpy>=1.14
-tqdm==4.19.9
+tqdm>=4.19
 demjson>=2.2
 torchtext>=0.2
 GitPython>=2.1
 PyYAML>=3.11
-pytest==3.3.2
+pytest==5.2.0
```

Comment:
Are these changes necessary for this PR? If not please remove it from this.

Reply:
I removed the changes with `>=` as they can be breaking in nature. Rest I accepted and then updated with our changes as of now.